routes/upload.js

```diff
@@ -54,7 +54,6 @@ app.post("/profileimage", authHelper.isLoggedIn, function(req, res) {
 	            	if (err) {
 	            		console.log(err);
```

Comment:
Let's add a line below with `res.sendStatus(500);` to indicate that there was an error saving the user.